games-action/openclonk-7.0

```diff
@@ -0,0 +1,128 @@
+SUMMARY="Free multiplayer action game"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you
+control\ clonks, small, but witty and nimble humanoid beings. The game
+is mainly about\
+mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+        openclonk$secondaryArchSuffix = $portVersion
+        app:OpenClonk$secondaryArchSuffix = $portVersion
+	appdata$secondaryArchSuffix = $portVersion
+    	appdata(openclonk.appdata.xml)$secondaryArchSuffix = $portVersion
+    	application$secondaryArchSuffix = $portVersion
+    	application$secondaryArchSuffix = $portVersion
```

Comment:
This is duplicated.
As i wrote earlier, only the binaries and libraries should be listed here.